taskcluster/ac_taskgraph/release_promotion.py

```diff
@@ -0,0 +1,163 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+
+from mozilla_version.maven import MavenVersion
+from taskgraph.actions.registry import register_callback_action
+
+from taskgraph.util.taskcluster import get_artifact
+from taskgraph.taskgraph import TaskGraph
+from taskgraph.decision import taskgraph_decision
+from taskgraph.parameters import Parameters
+from taskgraph.util.taskgraph import find_decision_task, find_existing_tasks_from_previous_kinds
+
+RELEASE_PROMOTION_PROJECTS = (
+    "https://github.com/mozilla-mobile/android-components", "https://github.com/escapewindow/android-components"
```

Comment:
Note to our future selves: Remove `https://github.com/escapewindow/android-components` once we're done with staging releases.